tools/test-js-with-node

```diff
@@ -61,6 +61,7 @@ EXEMPT_FILES = make_set(
         "static/js/click_handlers.js",
         "static/js/compose.js",
         "static/js/compose_actions.js",
+        "static/js/compose_banner.js",
```

Comment:
It seemed hard to write a reasonable test for a call to `set_scroll_to_message_banner_message_id`, and this file is also almost exclusively UI stuff that can't really be tested.